src/api/app/views/webui/project/edit.js.haml

```diff
@@ -0,0 +1 @@
+$('.in-place-edition').html("#{escape_javascript(render(partial: 'webui/project/edit', locals: { project: @project }))}");
```

Comment:
Why do you use the `.haml` pre-processor? There is not haml in here.

Reply:
True! There's no reason to use `.haml` to only render a partial. I'll use `.erb` instead.
For both edit and show.